content/en/observability_pipelines/guide/remap_reserved_attributes.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Remap Reserved Attributes
+disable_toc: false
+further_reading:
+- link: "observability_pipelines/processors/edit_fields/"
+  tag: "Documentation"
+  text: "Learn more about the Edit Fields processor"
+- link: "observability_pipelines/processors/custom_processor/"
+  tag: "Documentation"
+  text: "Learn more about the Custom Processor processor"
+---
+
+## Overview
+
+Observability Pipelines' processors enable you to add, edit, and remove log fields. Remapping attributes or rewriting values ensures your logs are processed and standardized properly. For the majority of processing use cases, use the Edit Fields processor to add, remap, or remove fields from your logs. For advanced use cases, use the Custom Processor to conditionally modify fields or rewrite a field's value.
+
+In Datadog, [reserved attributes][1] are log fields that are set aside for specific processing in the platform. Reserved attributes include ` host`, `source`, `status`, `service`, `trace_id`, and `message`. Reserved attributes are applied when routing logs to the following Observability Pipelines destinations:
+
+- Datadog Log Management
```

Comment:
Hi @urseberry! I committed your suggestion but then realized that the destination itself is called Datadog Logs in the UI and in the docs, so I'm going to change it back to that. 
<img width="293" alt="2025-07-08_12-36-21" src="https://github.com/user-attachments/assets/ea58830d-099a-41fc-9f57-624ccfc4e8fe" />
```suggestion
- Datadog Logs
```